griptape/engines/rag/stages/response_rag_stage.py

```diff
@@ -5,49 +5,35 @@
 
 from attrs import define, field
 
+from griptape import utils
 from griptape.engines.rag.stages import BaseRagStage
 
 if TYPE_CHECKING:
     from griptape.engines.rag import RagContext
     from griptape.engines.rag.modules import (
-        BaseAfterResponseRagModule,
-        BaseBeforeResponseRagModule,
         BaseRagModule,
         BaseResponseRagModule,
     )
 
 
 @define(kw_only=True)
 class ResponseRagStage(BaseRagStage):
-    before_response_modules: list[BaseBeforeResponseRagModule] = field(factory=list)
-    response_module: BaseResponseRagModule = field()
-    after_response_modules: list[BaseAfterResponseRagModule] = field(factory=list)
+    response_modules: list[BaseResponseRagModule] = field()
 
     @property
     def modules(self) -> list[BaseRagModule]:
         ms = []
 
-        ms.extend(self.before_response_modules)
-        ms.extend(self.after_response_modules)
-
-        if self.response_module is not None:
-            ms.append(self.response_module)
+        ms.extend(self.response_modules)
 
         return ms
```

Comment:
Do we still need this property?

Reply:
We do because we test for module name uniqueness by using this property.